08-communicating-results.Rmd

```diff
@@ -156,7 +156,7 @@ trust_gov_gt %>%
   print_gt_book(knitr::opts_current$get()[["label"]])
 ```
 
-We can add a few more enhancements, such as a title, a data source note, and a footnote with the question information, using the functions `tab_header()`, `tab_source_note()`, and `tab_footnote()`. If having the percentage sign in both the header and the cells seems redundant, we can opt for `fmt_number()` instead of `fmt_percent()` and scale the number by 100 with `scale_by = 100`.
+We can add a few more enhancements, such as a title (which is different from a caption), a data source note, and a footnote with the question information, using the functions `tab_header()`, `tab_source_note()`, and `tab_footnote()`. If having the percentage sign in both the header and the cells seems redundant, we can opt for `fmt_number()` instead of `fmt_percent()` and scale the number by 100 with `scale_by = 100`. The resulting table is displayed in Table \@ref(tab:results-table-gt2-tab)
```

Comment:
Do we need to add more context around who a title and caption are different?

Reply:
I added the following footnote but could be a sentence in direct test. 

> The function `tab_caption()` is intended for usage in R Markdown, Quarto, or bookdown to add the ability of cross-referencing where as the function `tab_header()` is used to add a title or subtitle to a table in any context including Shiny or GitHub flavor Markdown without cross-referencing.